src/Microsoft.Health.Fhir.Shared.Api/Features/Resources/Bundle/BundleHandler.cs

```diff
@@ -424,6 +424,34 @@ private async Task<EntryComponent> ExecuteRequests(Hl7.Fhir.Model.Bundle respons
                     {
                         HttpContext httpContext = request.HttpContext;
 
+                        string path = httpContext.Request.Path.Value;
```

Comment:
Is this change related to the Terminology service?

Reply:
This is from a bug-fix I did my first or second week. I will remove it and make a separate PR for it later.